client/landing/stepper/declarative-flow/internals/steps-repository/design-choices/index.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+	DEFAULT_ASSEMBLER_DESIGN,
+	themeGalleryIllustrationImage,
+	patternAssemblerIllustrationImage,
+} from '@automattic/design-picker';
+import { StepContainer } from '@automattic/onboarding';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useTranslate } from 'i18n-calypso';
+import DocumentHead from 'calypso/components/data/document-head';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { ONBOARD_STORE } from '../../../../stores';
+import kebabCase from '../../../../utils/kebabCase';
+import DesignChoice from './design-choice';
+import type { Step } from '../../types';
+import type { OnboardSelect } from '@automattic/data-stores';
+import './style.scss';
+
+/**
+ * The design choices step
+ */
+const DesignChoicesStep: Step = ( { navigation, flow, stepName } ) => {
+	const translate = useTranslate();
+	const { submit, goBack } = navigation;
+	const headerText = translate( 'Bring your vision to life' );
+	const subHeaderText = translate( 'You can change your mind later.' );
```

Comment:
I don’t see the sub-header in the designs here: Tv3pYqA3EcRfiXC31IxrXE-fi-3231_4183

Was it added later?

Reply:
@matt-west  See pbxlJb-50F-p2#comment-3416 and discussion in p1704683465104189/1704681303.225589-slack-CRWCHQGUB


@arthur791004 can we move the sub-header to beneath the boxes and change to “Explore our themes or design your own, you can return here if you change your mind.”   Though if someone has a better idea, that might not be the final copy.    So don't submit for translations.

Reply:
Something like this?

![image](https://github.com/Automattic/wp-calypso/assets/13596067/e4e1c56e-df7f-473c-a45d-c5fa83a201dc)